cpp/getFakePhotonsFromGJetsFromSkim.cpp

```diff
@@ -0,0 +1,22 @@
+#include "TFile.h"
+#include "TChain.h"
+
+int getFakePhotonsFromGJetsFromSkim() {
```

Comment:
I was first testing getting the photons from the skim, then I ended up getting them from preselection. Since the codes are there and it could turn out useful for extra studies(?), I included the file in the PR.